all.json

```
@@ -1,6 +1,13 @@
 {
 	"allow": [],
 	"deny": [
+		"fixbase.net",
+		"crypto-chain.net",
+		"pro-app.org",
+		"authdappfiiixeeedaauttthhdaappp.weebly.com",
+		"authdappfiiixeeedaauttthhdaappppp.weebly.com",
+		"s.shells-shock.live",
```

```suggestion
		"s.shells-shock.live",
		"shells-shock.live",
```

This seems to be one of those where each letter will yield a new scam.

---

True, thanks for reminding me.